docs/modeling/connectors/ootb/comprehend.md

```diff
@@ -0,0 +1,59 @@
+---
+Title: Comprehend connector
+---
+
+# Comprehend connector
+The Comprehend connector uses the [Amazon Comprehend service's](https://aws.amazon.com/comprehend/) natural language processing (NLP) to identify and analyze text from `UTF-8` plain text files.
+
+The `implementation` value of the Comprehend connector in a service task would be similar to the following:
+
+```xml
+<bpmn2:serviceTask id="ServiceTask_8tacy8mn" implementation="comprehendConnector.ENTITY" />
+```
+
+## Amazon Web Services (AWS) configuration
+The following are the Amazon Comprehend APIs that are called using the connector:
+
+* [Detect Dominant Language API](https://docs.aws.amazon.com/comprehend/latest/dg/API_DetectDominantLanguage.html)
+* [Detect Entities API](https://docs.aws.amazon.com/comprehend/latest/dg/API_DetectEntities.html)
+* [Batch Detect Entities API](https://docs.aws.amazon.com/comprehend/latest/dg/API_BatchDetectEntities.html)
+* [Start Entities Detection Job API](https://docs.aws.amazon.com/comprehend/latest/dg/API_StartEntitiesDetectionJob.html)
+* [Describe Entities Detection Job API](https://docs.aws.amazon.com/comprehend/latest/dg/API_DescribeEntitiesDetectionJob.html)
+
+The Comprehend connector requires an AWS account to access Amazon features. It also requires an [Identity and Access Management (IAM)](https://aws.amazon.com/iam/) user to have the `textract:DetectDocumentText` and `textract:AnalyzeDocument` permissions. 
+
+## Input parameters
+The following are the parameters that can be passed to the Comprehend connector as input parameters using the `ENTITY` action:
+
+| Parameter | Description | Type | Required? |
+| --------- | ----------- | ---- | --------- | 
+| `nodeId` | The node ID of the file to use from Alfresco Content Services | String | `*` |
+| `uri` | The URI of the file to use | String | `*` |
+| `files` | A [file](../../modeling/files.md) uploaded in a process and set as a process variable | File | `*` |
```

Comment:
Just to clarify. Files can be uploaded in a process (process files), but can also be uploaded in a form (by the end user) or produced by other connectors (e.g. Aspose connector)

Reply:
Thanks for the clarification - I've updated all 3 connectors to note about the additional sources